app/services/DR/requestManager.js

```diff
@@ -0,0 +1,183 @@
+import { env } from 'jsdom-jscore-rn';
+import { AsyncStorage } from 'react-native';
+import utmObj from 'utm-latlng';
+import fetch from '../../helpers/Fetch';
+const COV_CASES = 'https://corona.lmao.ninja/v2/countries/do';
+
+async function saveData(data, field) {
+  if (data) {
+    await AsyncStorage.setItem(`coviddr@${field}`, JSON.stringify(data));
+
+    return data;
+  } else {
+    return JSON.parse(await AsyncStorage.getItem(`coviddr@${field}`));
+  }
+}
+
+export async function getAllCases() {
+  // eslint-disable-next-line no-undef
+  let data;
+  fetch(`${COV_CASES}?endAt=${order - 10}`)
```

Comment:
I believe It should be like
```suggestion
  fetch(COV_CASES)
```

Reply:
got it